plexapi/client.py

```diff
@@ -199,7 +201,7 @@ def sendCommand(self, command, proxy=None, **params):
             self._last_call = t
         elif t - self._last_call >= 80 and self.product in ('ptp', 'Plex Media Player'):
             self._last_call = t
-            self.timeline(wait=0)
+            self.timelines()
```

Comment:
I'm not a fan of this change. This will slow down any commands. We should use a raw self.sendCommand('timeline/poll', wait=0) as we don't care about the response. If a user really want the timeline info they should self.timlines() or self.timeline